src/composables/approvals/useTokenApprovalActions.ts

```diff
@@ -89,6 +94,7 @@ export default function useTokenApprovalActions() {
     amountsToApprove: AmountToApprove[],
     spender: string
   ): Promise<AmountToApprove[]> {
+    await injectSpenders([spender]);
```

Comment:
Since `injectSpenders` triggers the allowance query and waits for it, could we not remove the `refetchAllowances` call after it? If we do, I would add a comment above saying that this is what we expect to happen, in case someone tries to remove that from the `injectSpenders` function in the future.

Reply:
True, will fix.